doc/user/content/get-started/install-materialize-emulator.md

```diff
@@ -62,7 +62,7 @@ not suitable for production deployments</redb>.
    been already downloaded.
 
    ```sh
-   docker run -d -p 6875:6875 -p 6876:6876 materialize/materialized:{{< version >}}
+   docker run -d -p 127.0.0.1:6875:6875 -p 127.0.0.1:6876:6876 materialize/materialized:{{< version >}}
```

Comment:
I'm actually fine with this.  But, I'll leave it to @ggnall to determine whether we want additional blurb or not. 

Also, https://materialize.com/download/ page also contains the docker run command.

Reply:
Added a short blurb below.